client/scss/components/_form-side.scss

```diff
@@ -20,11 +28,11 @@
       w-py-4
       w-bg-white
       w-box-border
-      w-transition
+      w-transition-all
+      motion-reduce:w-transition-none
       w-duration-300
       w-border-l
       w-border-grey-100
-      w-overflow-y-auto
```

Comment:
I think this will make it impossible to scroll the panels if they’re taller than the viewport. Not sure why this change was needed?

Reply:
This was needed so that the resize grip isn't cut off because it overflows the parent. I've moved the `w-overflow-y-auto` and paddings to `__panel` to fix the scrolling.